frontend/server/src/Controllers/User.php

```diff
@@ -1429,6 +1429,53 @@ public static function apiProfile(\OmegaUp\Request $r): array {
         );
     }
 
+    /**
+     * When logged user is not allowed to see the information of another user
```

Comment:
```suggestion
     * When the user is not allowed to see the information of another user
```

aquí importa si el usuario está loggueado o no?

Reply:
Hmm, creo que no.